rosetta-source/src/main/rosetta/event-common-func.rosetta

```diff
@@ -1661,6 +1661,87 @@ func Qualify_Shaping: <"The qualification of a shaping event from the fact that
 		and packageRef count = openTradeNoExecutionDetails count
 		and packageRef distinct count = 1
 
+
+func Create_PartialDeliveryPrimitiveInstruction: <"Creates the primitive instructions for a substitution of collateral by replacing the assetpayout of the trade.">
```

Comment:
If the function is related to the substitution of collateral - why is it called "PartialDelivery"? Generally perhaps a more fulsome definition of the function's purpose would help - ready through the steps and inputs alone it seems to do something quite sophisticated?
_E.g. this instruction is to split a trade in two and change the first bit this way and second bit that way to effect this change overall_